demo/index.html

```diff
@@ -793,6 +793,28 @@ <h3>USPS has updated their rates</h3>
 
     <hr>
 
+    <div class="row no-gutters">
+      <div class="col-md-4 card-annotation">
+        <h3 class="display-3">Toast
+          <a href="https://polaris.shopify.com/components/feedback-indicators/toast" class="btn btn-link btn-sm" target="_blank">Polaris usage guide</a>
+        </h3>
+      </div>
+      <div class="col-md-8">
+        <div class="card">
+          <div class="card-body">
+            <h3>Basic Toast</h3>
+            <button id='toastButton' type="button" class="btn" onclick="var show = this.innerHTML == 'Show Toast'; var obj = document.getElementById('sampleToast'); if(show) { obj.classList.add('show'); this.innerHTML = 'Hide Toast'; } else { obj.classList.remove('show'); this.innerHTML = 'Show Toast'; }">Show Toast</button>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="frame-toast-manager">
+      <div class="frame-toast" id='sampleToast'>Message sent<button onclick='this.parentElement.classList.remove("show"); document.getElementById("toastButton").innerHTML = "Show Toast"' type="button" class="frame-toast-close"><span class="polaris-icon"><svg viewBox="0 0 20 20" focusable="false" aria-hidden="true"><path d="M11.414 10l6.293-6.293a.999.999 0 1 0-1.414-1.414L10 8.586 3.707 2.293a.999.999 0 1 0-1.414 1.414L8.586 10l-6.293 6.293a.999.999 0 1 0 1.414 1.414L10 11.414l6.293 6.293a.997.997 0 0 0 1.414 0 .999.999 0 0 0 0-1.414L11.414 10z" fill-rule="evenodd"></path></svg></span></button></div>
```

Comment:
The close button needs some kind of text to make it accessible to screen reader users. Using the `sr-only` class to visually hide the text but have it still be read by screen readers.
Markup could also use a little formatting.

```suggestion
      <div class="frame-toast" id='sampleToast'>Message sent
        <button onclick='this.parentElement.classList.remove("show"); document.getElementById("toastButton").innerHTML = "Show Toast"' type="button" class="frame-toast-close">
          <span class="sr-only">Close Toast</span>
          <span class="polaris-icon"><svg viewBox="0 0 20 20" focusable="false" aria-hidden="true"><path d="M11.414 10l6.293-6.293a.999.999 0 1 0-1.414-1.414L10 8.586 3.707 2.293a.999.999 0 1 0-1.414 1.414L8.586 10l-6.293 6.293a.999.999 0 1 0 1.414 1.414L10 11.414l6.293 6.293a.997.997 0 0 0 1.414 0 .999.999 0 0 0 0-1.414L11.414 10z" fill-rule="evenodd"></path></svg></span>
        </button>
      </div>
```